docs/application-connector/docs/032-gs-create-remote-environment.md

```diff
@@ -3,99 +3,22 @@ title: Create a new Remote Environment
 type: Getting Started
 ---
 
-The Remote Environment Controller provisions and de-provisions the necessary deployments for the created Remote Environment (RE).
-
-The following operations are available:
-- Create a new Remote Environment
-- Delete a Remote Environment
-- Update the Remote Environment configuration
-
-You can perform all these operations using either the Command Line Interface (CLI) or the Console UI.
-
-The controller creates all Remote Environments in the `kyma-integration` Namespace.
+The Remote Environment Controller provisions and de-provisions the necessary deployments for the created Remote Environment (RE). The controller creates all Remote Environments in the `kyma-integration` Namespace.
```

Comment:
The Remote Environment Controller doesn't create REs, it deploys resources related to it. REs are not namespaced resource so they aren't in any namespace.

Reply:
fixed 
"The controller creates all Remote Environment Custom Resources in the `kyma-integration` Namespace."